index.html

```diff
@@ -0,0 +1,313 @@
+  statement</code>
+      </p>
+      <p>If the condition becomes false, statement within the loop stops executing and control passes to the statement following the loop.</p>
+      <p>The condition test occurs before statement in the loop is executed. If the condition returns true, statement is executed and the condition is tested again. If the condition returns false, execution stops and control is passed to the statement following while.</p>
+      <p>To execute multiple statements, use a block statement ({ ... }) to group those statements.</p>
+      <p>Example:</p>
+      <p>The following while loop iterates as long as n is less than three:<code>var n = 0;
+var x = 0;
+while (n &lt 3) {
+  n++;
+  x += n;
+}</code>
+      </p>
+      <p>With each iteration, the loop increments n and adds that value to x. Therefore, x and n take on the following values:
+        <ol>
+          <li>After the first pass: n = 1 and x = 1</li>
+          <li>After the second pass: n = 2 and x = 3</li>
+          <li>After the third pass: n = 3 and x = 6</li>
+        </ol>
+      </p>
+      <p>After completing the third pass, the condition n &lt 3 is no longer true, so the loop terminates.</p>
+      </article>
+    </section>
+
+    <section id="Function_declarations" class="main-section">
+      <header>
+        <h2>Function declarations</h2>
+      </header>
+      <article>
+      <p>A function definition (also called a function declaration, or function statement) consists of the function keyword, followed by:
+        <ol>
+          <li>The name of the function.</li>
+          <li>A list of arguments to the function, enclosed in parentheses and separated by commas.</li>
+          <li>The JavaScript statements that define the function, enclosed in curly brackets, { }.</li>
+        </ol>
+      </p>
+      <p>For example, the following code defines a simple function named square:<code>function square(number) {
+  return number * number;
+}</code>
+      </p>
+      <p>The function square takes one argument, called number. The function consists of one statement that says to return the argument of the function (that is, number) multiplied by itself. The return statement specifies the value returned by the function.<code>return number * number;</code>
+      </p>
+      <p>Primitive parameters (such as a number) are passed to functions by value; the value is passed to the function, but if the function changes the value of the parameter, this change is not reflected globally or in the calling function.</p>
+      </article>
+    </section>
+
+    <section id="Reference" class="main-section">
+      <header>
+        <h2>Reference</h2>
+      </header>
+      <article>
+      <p>All the documentation in this page is taken from <a href="https://developer.mozilla.org/en-US/docs/Web/JavaScript/Guide">MDN</a></p>
+      </article>
```

Comment:
[REQUIRED] When using <a> tag's it is important to think about accessibility. 

To make HTML acessible means considering screen readers, and that consists of creating e Alt="description of the link" inside of the <a> tags.

In this example, it would mean:

```
> <p>All the documentation in this page is taken from <a href="https://developer.mozilla.org/en-US/docs/Web/JavaScript/Guide" alt="Mozilla's Javascript Guide">MDN</a></p>
>       </article>
```

Also, here it is important to note that indentation is not properly set-up. As mentioned in other review notes. (Details of that can be found in the other comments).

Reply:
Hi, @cancelei!! 👋
- Well done! Here you have made a good suggestion with reasoning. 👍 

- [ ] Kindly phrase the comments politely as suggestions instead of orders.
  - For example:
    - After the suggestion you can add this: "I think it would be better if you solve the accessibility issue by following the given suggestion and the indentation using Linters mentioned above..". 
- [ ] Kindly use emoticon (😄) so that your review looks human-friendly.